src/layouts/Login/LoginPage.stories.tsx

```diff
@@ -0,0 +1,102 @@
+import { ComponentStory, ComponentMeta } from "@storybook/react"
+import { rest } from "msw"
+import { MemoryRouter, Route } from "react-router-dom"
+
+import { handlers } from "../../mocks/handlers"
+
+import { LoginPage } from "./LoginPage"
+
+const SEND_OTP_ENDPOINT = `${process.env.BACKEND_URL_V2}/auth/login`
+const VERIFY_OTP_ENDPOINT = `${process.env.BACKEND_URL_V2}/auth/verify`
+const LOGOUT_ENDPOINT = `${process.env.BACKEND_URL_V2}/auth/logout`
+
+const LoginPageMeta = {
+  title: "Pages/LoginPage",
+  component: LoginPage,
+  parameters: {
+    // Set delay so mock API requests will get resolved and the UI will render properly
+    chromatic: { delay: 500 },
+    msw: {
+      handlers,
+    },
+  },
+  decorators: [
+    (Story) => {
+      return (
+        <MemoryRouter initialEntries={["/"]}>
+          <Route path="/">
+            <Story />
+          </Route>
+        </MemoryRouter>
+        // </ServicesContext.Provider>
+      )
+    },
+  ],
+} as ComponentMeta<typeof LoginPage>
+
+const Template: ComponentStory<typeof LoginPage> = LoginPage
+
+export const Default = Template.bind({})
+Default.parameters = {
+  msw: {
+    handlers: [
+      rest.post(SEND_OTP_ENDPOINT, (req, res, ctx) => {
+        return res(ctx.json({}))
+      }),
+      rest.post(VERIFY_OTP_ENDPOINT, (req, res, ctx) => {
+        return res(ctx.json({}))
+      }),
+      ...handlers,
```

Comment:
should we use the utils in `src/mocks/utils.ts` to ease boilerplate?

Reply:
The utils seem to be for `get` requests, and we need to make new ones for `post`/`delete` - I think we should do this separately